packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -184,6 +188,7 @@ export class CeramicCliUtils {
       }
     }
 
+    handleHeapdumpSignal(new URL('./', configFilepath))
```

Comment:
isn't the configFilepath already an absolute path at this point?  Why are we adding `./` to the front of it here but not in `_loadDaemonConfig`?

Reply:
`configFilepath` is an absolute file path _to a config file_. We are interested in its parent folder, thus `./`.

Reply:
Ah got it, that makes sense.  Thanks

Reply:
Just in case, for `absolute = new URL('file:///hello/world.txt')`, `new URL('./', absolute)` would return `new URL('file:///hello/')`.

`./` means "get me the current folder". If  a URL references a file, i.e. no trailing slash, it means the parent folder. If the  URL references a folder, i.e. there is a trailing slash, it returns the same folder.